data/css/mesh.scss

```diff
@@ -49,6 +49,8 @@ $display-font-composite: 'Roboto' !default;
 
 @import 'modules/window/simple';
 
+@import 'widgets/tabButton';
```

Comment:
I wasn't suggesting to exclude the paginated scrolling import from the templates, neither to exclude the button widget styling import from paginated scrolling partial.

What I suggested was to import the button styling in hamburger basement partial, and then import the hamburger basement partial in the templates that use it (just like we do with the scrolling).

We would not lose the button styling in the section page because we are also importing it in the paginated scrolling partial, and if we change the front-page to some other module that also uses that button, then that other module partial should also import the button.

My point is, the button widget styling makes more sense inside the modules that use it, because technically the button is just an implementation detail of that module.